pool/web/view/page/page_admin_custom_fields.ml

```diff
@@ -0,0 +1,341 @@
+open Tyxml.Html
+open Component
+module Message = Pool_common.Message
+
+let base_path = "/admin/custom-fields"
+
+let form
+  ?custom_field
+  Pool_context.{ language; csrf; _ }
+  sys_languages
+  flash_fetcher
+  =
+  let open Custom_field in
+  let action =
+    match custom_field with
+    | None -> base_path
+    | Some f -> Format.asprintf "%s/%s" base_path (f.id |> Id.value)
+  in
+  let checkbox_element ?orientation ?help ?(default = false) field fnc =
+    checkbox_element
+      language
+      ?orientation
+      ?help
+      field
+      ~value:(custom_field |> CCOption.map_or ~default fnc)
+      ~flash_fetcher
+  in
+  let value = CCFun.flip (CCOption.map_or ~default:"") custom_field in
```

Comment:
Bad "function" name, there are fields called "value", very hard to spot where the function is called. I found only one spot, may include it direclty.